.github/workflows/release.yml

```diff
@@ -51,13 +49,11 @@ jobs:
         run: ls -l dist/
 
       - name: Publish to PyPI
-        if: >-
-          github.event.inputs.target == 'pypi' || (github.event_name ==
-          'release' && github.event.action == 'published')
+        if: github.event_name == 'release'
```

Comment:
I think this is missing that the event.action needs to be "published", or it will also fire on other actions.

Reply:
Isn't this still open?

Reply:
Fixed. I did it for `url` and `name` but forgot to here.